section9_dictionary/project/main.py

```diff
@@ -0,0 +1,34 @@
+from art import logo
+import os
+
+print(logo)
+print("Welcome to secret auction program.")
+istherebidders = True
+bidders = {}
+greatest_bid = 0
+
+
+while istherebidders:
+    
+    bidder_name = input("What is your name?: ").lower()                     
+    individual_bid = int(input("What is you bid?: $"))
+    bidders[individual_bid] = bidder_name                                               #Put each bidder and their bid into dictionaries name bidders 
```

Comment:
Using bid amount as a key and the bidder name as the value seems weird to me, should be done vice-versa